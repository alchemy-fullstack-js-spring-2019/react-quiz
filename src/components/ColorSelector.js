import React from 'react';
import PropTypes from 'prop-types';

function ColorSelector({ changeHandler }){
  return (
      <>
      <input name="color" type="color" onChange={changeHandler}/>
      <input name="backgroundColor" type="color" onChange={changeHandler}/>
      </>
  );
}


ColorSelector.propTypes = {
  changeHandler: PropTypes.func.isRequired
};

export default ColorSelector;
