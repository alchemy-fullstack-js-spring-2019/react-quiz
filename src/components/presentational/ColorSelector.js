import React from 'react';

export default function ColorSelector() {
  return (
    <input name="background-color" type="color" />,
    <input name="text-color" type="color" />
  );
}
