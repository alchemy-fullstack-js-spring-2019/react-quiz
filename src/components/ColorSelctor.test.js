import React from 'react';
import { shallow } from 'enzyme';
import ColorSelector from './ColorSelector';

describe('ColorSelector Component', () => {
  it('renders the color', () => {
    const wrapper = shallow(<ColorSelector />);
    expect(wrapper).toMatchSnapshot();
  });
});
