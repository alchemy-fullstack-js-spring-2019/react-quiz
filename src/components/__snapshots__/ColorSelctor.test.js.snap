// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorSelector Component renders the color 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    name="textColor"
    onChange={[Function]}
    type="color"
    value="#000001"
  />
  <input
    name="color"
    onChange={[Function]}
    type="color"
    value="#000000"
  />
</form>
`;
